optax/losses/segmentation.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
```

Comment:
I don't know about this loss but why can't it go to `classification.py`?

Reply:
We can think about segmentation as a classification of pixels. The loss expects the network output to be a map with class probabilities for each pixel. Having a segmentation file would stress that this loss is not meant for tasks like image classification, with per-class probabilities for the entire image.

Reply:
Sine the file is not visible to end-users in the documentation, I don't think my initial argument matters. I have moved the function into `classification.py`.

Reply:
Thanks. That was also my opinion.